submodules/power_management_services_setup/scripts/squid_proxy.sh

```diff
@@ -26,28 +26,29 @@ exit_abnormal() {                         # Function: Exit with error.
 ############################################################
 if [[ $# -eq 0 ]] ; then
     Help
-    exit 0
+    exit 1
 fi
 
 while getopts :h?:p:in: flag
 do
     case "${flag}" in
-		p)
-		proxy_ip_and_port=${OPTARG};;
+	  p)
+	    proxy_ip_and_port=${OPTARG};;
```

Comment:
can we be consistent with indentation here? It is a bit of a mess right now:

![image](https://user-images.githubusercontent.com/29608224/190184957-9bc28eff-eec4-4e4f-888b-c22b7869da90.png)